ping/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: Ping
+
+## Overview
+
+This check uses the system [ping][1] command to test the reachability of a host.
+It also optionally measures the round-trip time for messages sent the check to the
+destination host.
+
+Ping operates by sending Internet Control Message Protocol (ICMP) echo request packets
+to the target host and waiting for an ICMP echo reply.
+
+This check uses the system ping command, rather than generating the ICMP echo request
+itself, as creating an ICMP packet requires a raw socket, and creating raw sockets
+requires root privileges, which the agent does not have. The ping command uses the
+`setuid` access flag to run with elevated privileges, avoiding this issue.
+
+## Setup
+
+### Installation
+
+The Ping check is not included in the [Datadog Agent][2] package, so you will
```

Comment:
```suggestion
The Ping check is not included in the [Datadog Agent][2] package, so you
```

Reply:
^ Style guide, no future tense in docs.

Reply:
Fixed

Looks like this needs to be fixed in the template as well: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/create.py#L59